cfgov/unprocessed/apps/hmda/.yarnrc

```diff
@@ -1 +0,0 @@
-yarn-offline-mirror "./npm-packages-offline-cache"
```

Comment:
Say a `package.json` is added in the future to this app and yarn is run, would the developer be expected to add this `.yarnrc` file back at that point? And what happens if they don't?

Reply:
I think it's fair to expect the developer to know to re-add the .yarnrc file, yes.

Reply:
And if they don't re-add it, what happens? Does it use the npm cache at the repo root? Maybe we don't need these .yarnrc files in the apps at all? Perhaps a question for @wpears

Reply:
IIRC because of the way we treat these apps as kinda separate sub-apps with their own `yarn.lock`, the `.yarnrc` file is needed (when js is present) in order to trigger use of the offline cache. Without it, it won't default to the parent cache, and even trying to reference the parent cache is somewhat fraught/buggy from my testing.

Ideally, imo, these apps shouldn't really be treated as their own sub-apps from a build standpoint so that they can be better bundled/share code with the main bundle more completely. Such a change would also obviate the need for these sub-app config files.

In sum, I support the removal of this specific `.yarnrc` file, the maybe the hope that in the future we will have a simpler build system and can then remove all of them except the top-level one!

Reply:
> Ideally, imo, these apps shouldn't really be treated as their own sub-apps from a build standpoint so that they can be better bundled/share code with the main bundle more completely. Such a change would also obviate the need for these sub-app config files.

I think the one situation we don't want to be in, whatever the setup, is where there's a shared dependency between say the owning-a-home rate checker (complex app) and somewhere else on the site that is far simpler. Someone goes in and updates the dependency on the far simpler thing, and either the rate checker is overlooked or it "looks fine" because there isn't a clear product owner and it's unclear how it should function. Months later it's discovered it's not fine and the team is faced with untangling what went wrong, while simultaneously keeping in mind what's changed in the site elsewhere. Or the other direction where the simpler area of the site is blocked from being updated because of the changes needed in the rate checker to allow the update to happen.

Reply:
Yeah, I think that's a good argument for keeping them separate, Ans.